assignments/4 - bookly/app.js

```diff
@@ -0,0 +1,46 @@
+var bookArr = JSON.parse(localStorage.getItem('book-arr')) || [];
+var addBtn = document.querySelector('.addButton')
+addBtn.addEventListener('click', addBooks)
+document.querySelector('#hide').addEventListener('click', visibilityNone)
+var bookInput = document.querySelector('.bookInput')
+var searchInput = document.querySelector('.searchInput')
+window.addEventListener('keyup', (e)=>searchBooks(e))
+
+function addBooks (event){
+    event.preventDefault()
+    bookArr.push(bookInput.value)
+    console.log(bookArr)
+    bookInput.value= '';
+    display(bookArr)
+}
+
+function searchBooks (e) {
+    e.preventDefault()
+    if( e.keyCode == 13) {
+        var bookName = searchInput.value
+        var filterBook = bookArr.filter(book => book==bookName) 
```

Comment:
Its better to convert the search term and book name in lowercase first and filter.